packetbeat/beater/devices.go

```diff
@@ -1,39 +0,0 @@
-package beater
```

Comment:
it's mostly for sanity, as I wanted the `libbeat/cmd/instance` module to be used only internally. The `-devices` flag always has been a pretty ugly hack, generating some dependency between the beat and some weird global access to libbeat internals.